enterprise/coderd/prebuilds/metricscollector.go

```diff
@@ -157,6 +157,10 @@ func (mc *MetricsCollector) Collect(metricsCh chan<- prometheus.Metric) {
 			continue
 		}
 
+		if preset.Deleted {
```

Comment:
I'm assuming this is here because of the new comments in prebuilds.sql?

Reply:
Yes, correct.